### PR TITLE
Implicit dim for  log_softmax  is deprecated.

### PR DESCRIPTION
**Pull request by bapriddy**
Replace
``` return F.log_softmax(x)  ``` 
with 
``` return F.log_softmax(x, dim=1) ```

### CODE REVIEW

**Comment by bapriddy** [comment]
@alsrgv 

The mnist example uses F.log_softmax(x) call without dim=X.  This gives the following error...

``` /home/user/anaconda3/lib/python3.6/site-packages/ipykernel_launcher.py:25: UserWarning: Implicit dimension choice for log_softmax has been deprecated. Change the call to include dim=X as an argument. ```

It should probably be updated.

**Comment by alsrgv** [comment]
@bapriddy, thanks for your PR!  Could you sign the commit using [DCO](https://github.com/horovod/horovod/pull/885/checks?check_run_id=73148741)?

**Comment by alsrgv** [comment]
@bapriddy, you can sign using:
```
$ git rebase --signoff master
$ git push -f
```

**Comment by alsrgv** [comment]
@bapriddy, bump.

**Comment by abditag2** [comment]
@bapriddy thanks for the PR. can you please sign the commit using: https://github.com/horovod/horovod/pull/885/checks?check_run_id=73148741

**Comment by bapriddy** [comment]
@abditag2 Can I do it from the browser?  Or do I have to do a pull, commit, sign, push??

**Comment by abditag2** [comment]
You'd need to do it from terminal.

**Comment by bapriddy** [comment]
Okay Thanks!
